deb/build

```diff
@@ -28,6 +28,7 @@ trap "rm -f '$changelog'; exit 1" INT TERM QUIT
 pkgname=libdrizzle-redux
 genchangelog "$pkgname" "$pkgversion" "$pkgmaint" > "$changelog"
 fpm -s dir -t deb -n "$pkgname" -v "$pkgversion" \
+	--prefix=/ \
```

Comment:
fpm is quite silly, it will copy stuff wherever you tell it to copy them, I think that option is for other types of packages.